### PR TITLE
Update RevisionHistory.html for v7.2.2

### PR DESCRIPTION
**Pull request by atsju**
I propose that we do a release for 7.2.2 with recent fixes.

(the tag should be done on master after the merge)

### CODE REVIEW

**Comment by gr5** [comment]
I was thinking the same.  I'll look at your notes and suggest changes today.  I'll do the actual release step if you don't mind as I want to test it for at least 20 minutes before approving release.  I guess you could do everything but the final approval if you want.

File: RevisionHistory.html [hunk 1]
```diff
@@ -409,3 +408,13 @@
 <li>First version that isn't compatible with Windows 7</li>
 </ul>
 </ul>
+
+<ul><li>Version 7.2.2</li>
+<ul>
+<li>Improved DLL dependecies behavior for developpers</li>
```
Comment (gr5) [inline RevisionHistory.html:414]:
```suggestion
<li>Improved DLL dependencies behavior for developers</li>
```

File: RevisionHistory.html [hunk 2]
```diff
@@ -409,3 +408,13 @@
...
+<li>Fixed a bug in mirror config when switching between inch and millimeter</li>
+<li>Fixed a limitation in mirror config for very large mirrors</li>
+<li>Fixed a bug where defocus adjustment would make DFTFringe crash</li>
+<li>Improved saving speed when workin on network drives</li>
```
Comment (gr5) [inline RevisionHistory.html:418]:
```suggestion
<li>Improved saving speed when working on network drives</li>
```

Comment (gr5) [reply RevisionHistory.html:418]: Looks great.  Please fix the spelling errors and I'll approve the PR.

**Comment by atsju** [comment]
Sorry I should have read myself

**Review by gr5 — APPROVED** [review]

**Comment by gr5** [comment]
@atsju  I'll do the tag.  Okay?

**Comment by atsju** [comment]
> @atsju I'll do the tag. Okay?

Yes please

**Comment by gr5** [comment]
Ug.  While testing, I found another bug when reading mirror config files.  Let me fix it tomorrow.

**Pull request merged**